metadata-app/src/components/MetadataForm/index.js

```diff
@@ -155,6 +157,13 @@ const MetadataForm = (props) => {
                 .catch(handleError);
             }
           }}
+          validate={() => {
+            // Note this isn't validating anything. We use "validationSchema"
+            // property to validate form values against "yup" schema. This
+            // method is needed to make sure that alert box is cleared on
+            // validation so we don't confuse users with multiple alert boxes.
+            setAlert();
```

Comment:
Just curious, since you already added `setAlert()` in `onSubmit()`, do we still need to have it here as well?

Reply:
@kmanaseryan yes, that's the thing - the way formik works, it doesn't reach `onSubmit()` method if validation doesn't pass. I also couldn't find another way of updating the state just before or after validation against provided schema.

Reply:
OK I see, thanks